_pages/install/integrations/open-payment-framework-integration.md

```diff
@@ -0,0 +1,132 @@
+
+If you have not yet set up your Spartacus app, you use the Spartacus schematics to select `Open Payment Framework Integration` when choosing which features to include in your storefront app.
+
+If you already have a storefront project up and running, you can add the `opf` library to your project by running the following command from the root directory of your storefront app:
+
+```bash
+ng add @spartacus/opf
+```
+
+This command uses schematics to modify your application and add the modules that are needed to launch the `opf` library.
+
+### Open Payment Framework Library Subdivision
+
+The `opf` library is divided into two distinct sub-libraries ensure optimal performance and a seamless user experiences.
+
+The `opf` library is split into sub-libraries as follows:
+
+- **@spartacus/opf/base** contains the foundational elements of the Open Payment Framework integration. It provides the core functionalities that are essential for the library to operate.
+
+- **@spartacus/opf/checkout** is specifically tailored to functionalities that are associated with the checkout process in the Open Payment Framework. This sub-library is lazy-loaded only when a customer navigates to the checkout page. This lazy-loading ensures that the checkout functionalities are not loaded preemptively, which saves using resources until they are absolutely necessary.
+
+## Storefront Configuration
+
+The following sections describe how to configure Spartacus to work with the Commerce Cloud adapter, and how to configure the routing to work with Open Payment Framework.
+
+### Commerce Cloud Adapter Configuration
+
+An essential part of the Open Payment Framework architecture is the Commerce Cloud Adapter. To ensure that Spartacus can communicate with the Commerce Cloud Adapter, you need to set values for the `baseUrl` and the `commerceCloudPublicKey` properties, as shown in the following example:
+
+```ts
+provideConfig(<OpfConfig>{
+  opf: {
+    baseUrl: '<URL TO COMMERCE CLOUD ADAPTER>',
+    commerceCloudPublicKey: '<COMMERCE CLOUD PUBLIC KEY>',
+  },
+}),
+```
+
+The following is a description of the Commerce Cloud Adapter configuration properties:
+
+- `baseUrl` is the URL to the Commerce Cloud Adapter. This URL has the same domain as your workbench URL, which is provided to you by an SAP administrator. For more information, contact your SAP administrator for Open Payment Framework.
+
+- `commerceCloudPublicKey` is the public key provided by Open Payment Framework. This key is used to establish a connection between the storefront and the instance of SAP Commerce Cloud that provided this key. For more information about generating the `commerceCloudPublicKey`, see [Configure Client in Open Payment Framework Workbench](https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/0996ba68e5794b8ab51db8d25d4c9f8a/abd0dcd86a5d472e8fd6d22bff28e9c4.html?locale=en-US&state=DRAFT).
+
```

Comment:
I changed the wording (and meaning) somewhat of what is says here about `commerceCloudPublicKey`. I don't know if this is actually correct and/or useful information, but if I have gone too far in a wrong direction, just let me know! 😄

Reply:
Correct, this version is more informative than a previous one. We can keep it for sure.

Reply:
Perfect! Thank you very much 😊